bin/mozilla-en-US.py

```diff
@@ -0,0 +1,131 @@
+#!/usr/bin/env python
+"""This script creates or updates en-US repositories from Mozilla source
+   code to use for Mozilla product localization."""
+
+import os
+import shutil
+import subprocess
+
+
+TARGET_REPOS = {
+    'firefox-aurora': {
+        'folders': [
+            'browser', 'browser/branding/official', 'dom', 'netwerk',
+            'security/manager', 'services/sync', 'toolkit', 'webapprt',
+        ],
+        'repository': 'mozilla-aurora',
+    },
+    'firefox-for-android-aurora': {
+        'folders': ['mobile', 'mobile/android', 'mobile/android/base'],
+        'repository': 'mozilla-aurora',
+    },
+    'thunderbird-aurora': {
+        'folders': [
+            'chat', 'editor/ui', 'mail',
+            'other-licenses/branding/thunderbird'
+        ],
+        'repository': 'comm-aurora',
+    },
+    'lightning-aurora': {
+        'folders': ['calendar'],
+        'repository': 'comm-aurora',
+    },
+    'seamonkey-aurora': {
+        'folders': ['suite'],
+        'repository': 'comm-aurora',
+    },
+}
+
+
+SOURCE_REPOS = set(v["repository"] for v in TARGET_REPOS.values())
+
+
+def execute(command, cwd=None):
+
```

Comment:
Wait, what? SublimeLinter wants you to put spaces after the `def` line? I understood pep8 to be saying that you should surround the entire function definition (including the body of it) rather than the line with `def`.